modules/apps/transfer/types/msgs_test.go

```diff
@@ -54,22 +54,27 @@ func TestMsgTransferValidation(t *testing.T) {
 		msg     *types.MsgTransfer
 		expPass bool
```

Comment:
we should change these to `expError`. Since transfer is really important and we are changing things, it'd be nice to feel extra good about our test cases and make sure they are actually hitting the right checks